packaging/standalone/src/main/distribution/shell-scripts/bin/Neo4j-Management/Get-Java.ps1

```diff
@@ -146,47 +151,28 @@ Function Get-Java
     if ($javaPath -eq '') { Write-Error "Unable to determine the path to java.exe"; return $null }
     if ($javaCMD -eq '') { $javaCMD = "$javaPath\bin\java.exe" }
     if (-not (Test-Path -Path $javaCMD)) { Write-Error "Could not find java at $javaCMD"; return $null }
-  
+ 
     $ShellArgs = @()
 
+    Write-Verbose "Java detected at '$javaCMD'"
+    Write-Verbose "Java version detected as '$javaVersion'"
```

Comment:
Yeah, there's a "bug" in Java on windows and I haven't worked on how to get around it.  When you run 'java -version' it doesn't write it to STDOUT, it writes to STDERR :-1:  So getting the actual version is difficult if it's not in the registry.

http://stackoverflow.com/questions/13483443/why-does-java-version-go-to-stderr